<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>400</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QPushButton" name="pStart_btn">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>230</y>
      <width>99</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>start</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pStop_btn">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>230</y>
      <width>99</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>stop</string>
    </property>
   </widget>
   <widget class="QTextBrowser" name="pMsg_Txt">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>50</y>
      <width>341</width>
      <height>141</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="pSpo2_btn">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>280</y>
      <width>99</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>spo2</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pExit_btn">
    <property name="geometry">
     <rect>
      <x>490</x>
      <y>30</y>
      <width>99</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>exit</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pNibp_btn">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>280</y>
      <width>99</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>nibp</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>600</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
